examples/sundials/CMakeLists.txt

```diff
@@ -11,13 +11,15 @@
 
 set(SUNDIALS_EXAMPLES_SRCS)
 list(APPEND SUNDIALS_EXAMPLES_SRCS
+  test.cpp
   ex9.cpp
```

Comment:
while I know this isn't an extension of a normal MFEM test, a more standard name (`ex...`) would be apprecited.

Reply:
I agree with Jamie, a standard naming should be utilized.

Reply:
I will just pick a number that doesn't correspond to the MFEM examples then? Is this okay? @tzanio What do you suggest?